assets/airports/eddm.json

```diff
@@ -1,883 +1,2695 @@
+        "RENLO":    ["N48d46m50.00",  "E011d03m72.00"],
+        "AKANU":    ["N49d03m05.00",  "E010d39m48.00"],
+        "BURAM":    ["N48d41m73.00",  "E010d56m93.00"],
+        "ANORA":    ["N48d56m97.00",  "E010d32m90.00"],
+        "GESLU":    ["N48d27m72.00",  "E009d41m32.00"],
+        "KUNOD":    ["N48d20m33.00",  "E009d37m12.00"],
+        "PELOG":    ["N48d13m68.00",  "E009d40m50.00"],
+        "REDVO":    ["N48d39m45.00",  "E010d41m60.00"],
+        "RIXED":    ["N48d49m77.00",  "E010d25m12.00"],
+        "USEMI":    ["N49d06m52.00",  "E012d58m12.00"],
+        "LITLI":    ["N48d57m87.00",  "E012d51m83.00"],
+        "OSTES":    ["N48d42m37.00",  "E012d40m73.00"],
+        "ARBAX":    ["N49d12m67.00",  "E013d02m58.00"],
+        "ODUDA":    ["N49d04m48.00",  "E012d44m83.00"],
+        "NETES":    ["N48d54m38.00",  "E012d37m58.00"],
+        "UNKOM":    ["N48d47m80.00",  "E012d32m90.00"],
+        "MITNU":    ["N48d42m12.00",  "E012d29m68.00"],
+        "BAGMI":    ["N49d13m60.00",  "E012d51m43.00"],
+        "BAMTA":    ["N48d43m93.00",  "E012d53m25.00"],
+        "RDG"  :    ["N49d02m42.00",  "E012d31m59.00"],
+        "GOMAX":    ["N48d47m42.00",  "E012d20m92.00"],
+        "VESIX":    ["N48d41m60.00",  "E012d18m67.00"],
+        "EXUSI":    ["N49d13m37.00",  "E012d39m43.00"],
+        "DM472":    ["N48d07m20.00",  "E011d27m49.00"],
+        "DM473":    ["N48d07m76.00",  "E011d34m46.00"],
+        "DM474":    ["N48d08m71.00",  "E011d46m60.00"],
+        "DM475":    ["N48d09m17.00",  "E011d52m68.00"],
+        "DM476":    ["N48d10m09.00",  "E012d04m83.00"],
+        "DM457":    ["N48d16m63.00",  "E012d03m74.00"],
+        "DM456":    ["N48d16m18.00",  "E011d57m65.00"],
+        "DM455":    ["N48d15m72.00",  "E011d51m56.00"],
+        "DM454":    ["N48d15m25.00",  "E011d45m47.00"],
+        "DM453":    ["N48d14m78.00",  "E011d39m39.00"],
+        "DM452":    ["N48d14m30.00",  "E011d33m31.00"],
+        "DM451":    ["N48d13m29.00",  "E011d20m74.00"],
+        "DM450":    ["N48d12m56.00",  "E011d11m85.00"],
+        "DM458":    ["N48d17m42.00",  "E012d14m42.00"],
+        "DM459":    ["N48d18m15.00",  "E012d24m55.00"],
+        "REDBU":    ["N47d57m35.00",  "E012d49m10.00"],
+        "AMADI":    ["N47d57m35.00",  "E012d49m10.00"],
+        "DM466":    ["N48d05m48.00",  "E012d54m83.00"],
+        "DM465":    ["N48d07m52.00",  "E012d05m25.00"],
+        "DM464":    ["N48d06m61.00",  "E011d53m11.00"],
+        "DM463":    ["N48d06m14.00",  "E011d47m04.00"],
+        "LULAR":    ["N48d05m20.00",  "E011d34m91.00"],
+        "VESUB":    ["N49d21m95.00",  "E012d45m53.00"],
+        "BIGNU":    ["N49d25m16.00",  "E012d59m85.00"],
+        "NIRGO":    ["N49d19m86.00",  "E013d07m93.00"],
+        "PIGAB":    ["N49d11m90.00",  "E010d08m60.00"],
+        "DKB"  :    ["N49d08m57.00",  "E010d14m30.00"],
+        "LEVBU":    ["N49d00m42.00",  "E010d27m40.00"],
+        "ELMOX":    ["N49d23m02.00",  "E009d49m48.00"]
```

Comment:
Unless there's a significant advantage to keeping these grouped as they are, might as well sort them alphabetically (and check for duplicates!). :smile: